go.mod

```diff
@@ -258,9 +260,16 @@ replace (
 	// Use cosmos keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 
+	// TODO: remove this replace directive once the following PR is merged and released: https://github.com/CosmWasm/wasmd/pull/2139
+	github.com/CosmWasm/wasmd => github.com/gjermundgaraba/wasmd v0.0.0-20250227212003-93178b50a03e
+
 	// Use special SDK v0.50.x release with support for both ICS and LSM
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.50.11-lsm
 
+	github.com/cosmos/ibc-go/modules/light-clients/08-wasm/blsverifier => github.com/cosmos/ibc-go/modules/light-clients/08-wasm/blsverifier v0.0.0-20250211202654-9c448bbbe4ab
```

Comment:
Do you have to do this? If so, that might be an issue we need to fix.

Reply:
There was as point at which this errored if I didn't include it, but that doesn't seem to be the case now. Removed it.